test/upgrade/prober/wathola/event/services.go

```diff
@@ -28,15 +28,19 @@ var lastProgressReport = time.Now()
 
 // ErrorStore contains errors that was thrown
 type ErrorStore struct {
-	state  State
-	thrown []thrown
+	state            State
+	missingThrown    []thrown
+	duplicatedThrown []thrown
+	unexpectedThrown []thrown
 }
 
 // NewErrorStore creates a new error store
 func NewErrorStore() *ErrorStore {
 	return &ErrorStore{
-		state:  Active,
-		thrown: make([]thrown, 0),
+		state:            Active,
+		missingThrown:    make([]thrown, 0),
+		duplicatedThrown: make([]thrown, 0),
+		unexpectedThrown: make([]thrown, 0),
```

Comment:
Again, here also I would name it simply:

```golang
thrown: thrownTypes{
  missing:    make([]thrown, 0)
  duplicated: make([]thrown, 0),
  unexpected: make([]thrown, 0),
}
```

Then accessing it is much nicer: `f.errors.unexpectedThrown` -> `f.errors.thrown.unexpected`

Reply:
Fixed as suggested.